build.sbt

```diff
@@ -13,3 +13,22 @@ scalacOptions in Test ++= Seq("-deprecation")
 testOptions in Test += Tests.Argument("-Djava.library.path=target/native")
 
 publishTo := Some(Resolver.file("file",  new File( "releases" )) )
+
+lazy val packageSqlite3C = taskKey[Unit]("Packages only Sqlite3C.java")
+
+packageSqlite3C := {
+  import scala.sys.process._
```

Comment:
You don't need this in SBT. It has its own wrapper around process builder that's auto-imported.